Dockerfile

```diff
@@ -1,8 +1,5 @@
 FROM hseeberger/scala-sbt:latest
 
-RUN apt-get update -qq
-RUN apt-get install -y postgresql
```

Comment:
Is Postgres definitely part of the image? Or why can we safely remove it here?

Reply:
This is the command line postgres client that was used in start-marketplace.sh